pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -242,10 +242,50 @@ func StartVirtlog(stopChan chan struct{}) {
 
 			err := cmd.Start()
 			if err != nil {
-				log.Log.Reason(err).Error("failed to start libvirtd")
+				log.Log.Reason(err).Error("failed to start virtlogd")
 				panic(err)
 			}
 
+			go func() {
+				logfile := fmt.Sprintf("/var/log/libvirt/qemu/%s.log", domainName)
```

Comment:
I think that it is possible to configure virtlogd to directly print to stdout. Would it be possible to read them directly from virtlogd?

Reply:
@rmohr @victortoso Following my conversation with Daniel Berrange, the virtlogd can log to stderr only about it's own activity. Qemu logs will continue to be written to /var/log/libvirt/qemu. Even though we configure stdio_handler=logd in qemu.conf this pipe is only between qemu and virtlogd. As per my understanding from Daniel we cannot make virtlogd to redirect the incoming QEMU logging to something other than a file in /var/log/libvirt/qemu

Reply:
Fine by me then. That explains why my configuration attempts in the past had no influence.

Reply:
Yep, same with me. I was about to send an email to libvirt folks to see what I was doing wrong. Thanks @enp0s3 for the quick check with Daniel.